x-pack/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -135,6 +135,7 @@ export function TransactionDistribution({
   const timeFormatter = getDurationFormatter(xMax);
 
   const tooltipProps: SettingsSpec['tooltip'] = {
+    placement: 'bottom',
```

Comment:
Nice, is it then fixed to the bottom, so it doesn't move relative to the y-axis? If that's the case we should apply this to all the charts. The tooltip should always be fixed relative to the y-axis. 
However, instead of being fixed to the bottom, can we fix it to the middle?

Reply:
Turns out this doesn't fix the tooltip the way I hoped it did.